src/app/(with-header)/activities/[id]/components/ImageGrid.tsx

```
@@ -3,44 +3,83 @@
+              priority
+            />
+          </motion.div>
+        </AnimatePresence>
+
         <button
           onClick={prevSlide}
           aria-label='이전 이미지'
           className='absolute top-1/2 left-2 -translate-y-1/2 rounded-full bg-black/50 px-6 py-10 text-white'
         >
           ‹
         </button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)